src/main/java/com/direwolf20/buildinggadgets/common/tools/InventoryManipulation.java

```diff
@@ -113,16 +118,32 @@ public static boolean useItem(ItemStack itemStack, EntityPlayer player, int coun
     }
 
```

Comment:
Would a general interface make sense, which would then be implemented by some kind of wrapper for the world (save?) and the RemoteInventoryCache?

Reply:
Implemented.